src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -308,6 +308,8 @@ def helm_install_release(chart_path, subscription_id, kubernetes_distro, kuberne
                         "--set", "systemDefaultValues.spnOnboarding=false",
                         "--set", "global.azureEnvironment={}".format(cloud_name),
                         "--set", "systemDefaultValues.clusterconnect-agent.enabled=true",
+                        "--namespace", "{}".format(consts.Release_Install_Namespace),
```

Comment:
add comment over this function, not to be used for re-put scenarios

Reply:
check auto update logic if it takes --release-namespace correctly

Reply:
Verified this. Autoupdate is picking up release namespace and it was able to upgrade successfully